src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -132,10 +142,50 @@ public void mouseClicked(MouseEvent me) {
 		public void mouseEntered(MouseEvent me) {}
 
 		public void mouseExited(MouseEvent me) {}
+		
+		private Point draggingPos;
```

Comment:
Move this variable declaration up so that it is before any method definitions. It's better programming practice to have all variables declared in the same spot.